src/zenkit/_native.py

```diff
@@ -11,6 +11,12 @@
 
 
 class ZkString(c_char_p):
+    _encoding = "windows-1252"
```

Comment:
I'm not sure if ZenKit4Py intends to allow to read multiple DAT files at once. If that's the case, this approach is wrong, because ZkString is global for all `DaedalusScript` instances, so changing the encoding for one could break the other.

Reply:
Ah, you were faster at the review, than me at writing it 😆

Reply:
I removed the encoding part for now. There are over 160 occurrences of the magic string "windows-1252", so a lot of files will be changed at once, so I might just create another PR after this one 👌